db/schema.rb

```diff
@@ -500,14 +500,14 @@
     t.integer "pcodenk"
     t.string "postcode_full"
     t.boolean "is_la_inferred"
+    t.integer "hodate_check"
     t.bigint "bulk_upload_id"
     t.integer "retirement_value_check"
-    t.integer "hodate_check"
-    t.integer "extrabor_value_check"
-    t.integer "deposit_and_mortgage_value_check"
     t.integer "shared_ownership_deposit_value_check"
+    t.integer "extrabor_value_check"
     t.integer "grant_value_check"
     t.integer "old_persons_shared_ownership_value_check"
+    t.integer "deposit_and_mortgage_value_check"
     t.index ["bulk_upload_id"], name: "index_sales_logs_on_bulk_upload_id"
     t.index ["created_by_id"], name: "index_sales_logs_on_created_by_id"
     t.index ["owning_organisation_id"], name: "index_sales_logs_on_owning_organisation_id"
```

Comment:
Does this file need to be here?

Reply:
I thought that in general if any migrations were required (e.g. I try to run the app and it tells me to run a migration) then I should commit any changes due to that migration. Is that not true? Thanks!